src/layer/marker/Marker.js

```diff
@@ -202,9 +202,13 @@ export var Marker = Layer.extend({
 		var options = this.options,
 		    classToAdd = 'leaflet-zoom-' + (this._zoomAnimated ? 'animated' : 'hide');
 
+		var button = DomUtil.create('button', 'leaflet-marker-button');
 		var icon = options.icon.createIcon(this._icon),
 		    addIcon = false;
 
+		button.setAttribute('aria-haspopup', 'true');
+		button.setAttribute('aria-describedby', 'popup-tooltip');

```

Comment:
How is `aria-describedby` supposed to work here, if popups (why are popups called tooltips in this PR btw? Am I misunderstanding this PR?) are not visible to the user? In fact, popups are not even in the DOM for reference until they're already opened.